scripts/src/client.ts

```diff
@@ -320,18 +319,22 @@ export class Client {
 		}
 	}
 
-	public async submitOrder(orderId: string, content?: string): Promise<Order> {
+	public async submitOrder(orderId: string, options: {content?: string, transaction?: string} = {}): Promise<Order> {
```

Comment:
i'd do it like this:
```
public async submitOrder(orderId: string, options: { content: string } = {}): Promise<Order>;
public async submitOrder(orderId: string, options: { transaction: string } = {}): Promise<Order>;
public async submitOrder(orderId: string, options: { content?: string, transaction?: string } = {}): Promise<Order> {
    ...
}
```
that way when using this it's clear that i must pass either `{ content: "..." }` or `{ transaction: "..." }`